libs/components/forms/src/lib/modules/radio/radio.component.spec.ts

```diff
@@ -53,6 +53,20 @@ describe('Radio component', function () {
       expect(onDisabledChangeSpy).toHaveBeenCalledTimes(1);
     });
 
+    it('should emit when radio is checked', async () => {
+      const onCheckedChangeSpy = spyOn(testComponent, 'onCheckedChange');
+      const radios: NodeListOf<HTMLInputElement> =
+        fixture.nativeElement.querySelectorAll('input');
+
+      // Select the second radio.
+      radios.item(1).click();
+      fixture.detectChanges();
+      await fixture.whenStable();
+
+      expect(onCheckedChangeSpy).toHaveBeenCalledWith(false);
```

Comment:
You can combine these two into `toHaveBeenCalledOnceWith()`.

Reply:
I didn't know about that one. Cool!